common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1209,7 +1214,12 @@
          * Time out for the AccountManager's remove account operation in broker.
          */
         public static final int ACCOUNT_MANAGER_REMOVE_ACCOUNT_TIMEOUT_IN_MILLISECONDS = 5000;
-        
+
+        /**
+         * The Bundle key name of serialized parameters for the PoP auth scheme.
+         */
+        public static final String AUTHSCHEME_PARAMS_POP = "pop_parameters";
```

Comment:
```suggestion
        public static final String AUTH_SCHEME_PARAMS_POP = "pop_parameters";
```

Reply:
Applying this suggestion, but requires more changes than this -- so doing in
f6b06016dcda4d5a0cfbbadaac7af301adc584ec